bin/main.ml

```diff
@@ -166,6 +170,14 @@ let common =
                     build a particular $(b,<package>.install) target.|}
         )
   in
+  let enable_optional_pps =
+    Arg.(value
+         & opt (some (list string)) None
+         & info ["enable-optional-pps"] ~docs ~docv:"PACKAGES"
```

Comment:
`PACKAGES` is not very clear here, as we usually don't talk about findlib packages with jbuilder. Let's just use `PPX-REWRITERS`